src/main/java/org/broadleafcommerce/vendor/amazon/s3/S3FileServiceProvider.java

```diff
@@ -80,15 +90,42 @@ public File getResource(String name) {
 
     @Override
     public File getResource(String name, FileApplicationType fileApplicationType) {
-        File returnFile = blFileService.getLocalResource(buildResourceName(name));
+        File returnFile = blFileService.getLocalResource(getResourceName(name));
         OutputStream outputStream = null;
         InputStream inputStream = null;
 
         try {
             S3Configuration s3config = s3ConfigurationService.lookupS3Configuration();
             AmazonS3Client s3 = getAmazonS3Client(s3config);
-            S3Object object = s3.getObject(new GetObjectRequest(s3config.getDefaultBucketName(), buildResourceName(name)));
-            
+            String resourceName = getResourceName(name);
```

Comment:
@regcole it seems like you're just finding this value and throwing it away. Isn't what you really want the `getPossibleResourceNames()` that you have below? It seems like this doesn't do anything, along with the `StringUtils.isEmpty(resourceName)`; I can't figure out the function, since `getPossibleResourceNames()` will return the site-specific resource name along with the parent's resource name